polymetis/polymetis/python/scripts/robopen_dashboard_demos/sinusoidal_movement.py

```diff
@@ -0,0 +1,68 @@
+import a0
+import polymetis
+import numpy as np
+import time
+import serdes
+import json
+
+if __name__ == "__main__":
+    sensor_msgs = serdes.get_capnp_msgs("sensor_msgs")
+    joint_pub = a0.Publisher("robot/desired_joint_state/sinusoidal")
+    json_pub = a0.Publisher("robot/desired_joint_state/sinusoidal_json")
+
+    robot = polymetis.RobotInterface(ip_address="100.97.47.78", enforce_version=False)
+    robot.go_home()
+
+    control_hz = 50  # update frequency
+    T = 2  # starting period
+    time_left = 1000
+    chirp = 2500
+
+    target_joint = 4
+    joint_limit = 0.5
+    joint_positions = robot.get_joint_positions()
+
+    # robot.metadata.default_Kx[1] = 1000 # tuning gains
+    robot.start_joint_impedance()
+
+    for i in range(time_left * control_hz):
+        start = time.time()
+
+        # sinusoidal function
+        joint_positions[target_joint] = (
+            np.sin(np.pi * i / (T * control_hz)) * joint_limit
+        )
+
+        # chirp function
+        # joint_positions[target_joint] = np.sin(np.pi * i * i / (T * control_hz * chirp)) * joint_limit
+
```

Comment:
Can you either [define a constant](https://www.pythontutorial.net/python-basics/python-constants/) outside of `__main__` or create an argument using something like [argparse](https://docs.python.org/3/library/argparse.html) to switch between fixed-frequency sinusoidal and chirp? Either one works and is better than commenting out code.

Reply:
Yep, fixed! Thanks!